### ISSUE TITLE
Update Provider in MariaDBConsumer in shared-to-shared-migrate-k8s.sh

### ISSUE THREAD

**Issue by Schnitzel** [amazeeio/helper-scripts#2]
The script in https://github.com/amazeeio/helper-scripts/blob/main/shared-to-shared-migrate-k8s.sh is great to migrate from one MariaDBProvider to another. 
Unfortunately it does not change the MariaDBConsumer object after it has migrated the DB, to the dbaas-operator still thinks it's using the old MariaDBProvider.

Is there any easy way to change this? Just modify the MariaDBConsumer Object with the correct data? Or will this freak out the dbaas-operator?

**Comment by shreddedbacon** [comment]
Yep, just modifying the consumer with the provider information should be enough.
